webapp/components/ContentMenu.vue

```diff
@@ -43,7 +43,13 @@ export default {
         return value.match(/(contribution|comment|organization|user)/)
       },
     },
-    callbacks: { type: Object, required: true },
+    modalsData: {
+      type: Object,
+      required: false,
+      // default: () => {
```

Comment:
> [<img alt="Kachulio1" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Kachulio1) **Authored by [Kachulio1](https://github.com/Kachulio1)**
_<time datetime="2019-06-14T10:34:39Z" title="Friday, June 14th 2019, 12:34:39 pm +02:00">Jun 14, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -43,7 +43,13 @@ export default {
         return value.match(/(contribution|comment|organization|user)/)
       },
     },
-    callbacks: { type: Object, required: true },
+    modalsData: {
+      type: Object,
+      required: false,
+      // default: () => {
```


We can remove the commented out code.

Reply:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2019-06-14T16:40:59Z" title="Friday, June 14th 2019, 6:40:59 pm +02:00">Jun 14, 2019</time>_

---


This is not related to me. @Kachulio1